src/device.rs

```diff
@@ -513,17 +512,16 @@ impl<B: UsbBus> UsbDevice<'_, B> {
                     };
 
                     if let Some(s) = s {
-                        accept_writer(xfer, |w| w.string(s));
+                        accept_writer(xfer, |w| w.string(s))
                     } else {
-                        xfer.reject().ok();
+                        xfer.reject()
                     }
                 }
             }
 
-            _ => {
-                xfer.reject().ok();
-            }
+            _ => xfer.reject(),
         }
+        .unwrap();
```

Comment:
I don't think `unwrap()`ing the result here is the best approach. This will inject `fmt` bloat into applications for people that may not be able to pay the memory overhead.

We should likely bubble the error up instead and let the user handle the `Result`

Reply:
:+1: